uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -698,4 +723,17 @@ private MqttProtoVersion convertMqttVersion(String mqttVersion) {
 
         return version;
     }
+
+
+    @Data
+    @AllArgsConstructor
+    private class MqttBrokerConnection {
```

Comment:
That is not a connection, it just address information describe where is broker and which CA it has.
Please rename.

Reply:
fixed 8e80651166627c92920165de239a88a247c7d4a5